src/server/server_family.cc

```diff
@@ -869,6 +869,19 @@ void PrintPrometheusMetrics(const Metrics& m, StringResponse* resp) {
                       &db_key_expire_metrics);
   }
 
+  if (!m.replication_metrics.empty()) {
```

Comment:
Can we test this functionality in any way? Either unit tests or pytests?

Reply:
We should really do that.. We have no coverage for the metrics at all right now.